zhaquirks/xiaomi/aqara/remote_h1.py

```diff
@@ -0,0 +1,228 @@
+                    AqaraRemoteManuSpecificCluster,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    OnOff.cluster_id,
+                ],
+            }
+        }
+    }
+    device_automation_triggers = {
+        # triggers when operation_mode == event
+        # the button doesn't send an release event after hold
+        (SHORT_PRESS, BUTTON): {COMMAND: COMMAND_1_SINGLE},
+        (DOUBLE_PRESS, BUTTON): {COMMAND: COMMAND_1_DOUBLE},
+        (TRIPLE_PRESS, BUTTON): {COMMAND: COMMAND_1_TRIPLE},
+        (LONG_PRESS, BUTTON): {COMMAND: COMMAND_1_HOLD},
+        # triggers when operation_mode == command
+        (ALT_SHORT_PRESS, BUTTON): {COMMAND: COMMAND_TOGGLE, ENDPOINT_ID: 1, ARGS: []},
+        (ALT_DOUBLE_PRESS, BUTTON): {COMMAND: COMMAND_OFF, ENDPOINT_ID: 1, ARGS: []},
+    }
+
+
+class RemoteH1DoubleRocker(XiaomiCustomDevice):
+    """Aqara H1 Wireless Remote Double Rocker Version WRS-R02"""
+
+    signature = {
+        MODELS_INFO: [(LUMI, "lumi.remote.b28ac1")],
+        ENDPOINTS: {
+            1: {
+                # SizePrefixedSimpleDescriptor(
+                #   endpoint=1, profile=260, device_type=259, device_version=1,
+                #   input_clusters=[0, 3, 1], output_clusters=[3, 6])
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT_SWITCH,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
+                    Identify.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    OnOff.cluster_id,
+                ],
+            },
+            3: {
+                # SizePrefixedSimpleDescriptor(
+                #   endpoint=3, profile=260, device_type=259, device_version=1,
+                #   input_clusters=[3], output_clusters=[6])
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT_SWITCH,
+                INPUT_CLUSTERS: [Identify.cluster_id],
+                OUTPUT_CLUSTERS: [OnOff.cluster_id],
+            },
+        },
+    }
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT_SWITCH,
+                INPUT_CLUSTERS: [
+                    BasicCluster,
+                    Identify.cluster_id,
+                    PowerConfigurationClusterH1Remote,
+                    MultistateInputCluster,
+                    AqaraRemoteManuSpecificCluster,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    OnOff.cluster_id,
+                ],
+            },
+            2: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.ON_OFF_LIGHT_SWITCH,
```

Comment:
I'm not entirely sure which `DEVICE_TYPE` should i choose here. At least it seems this one works.